packages/server/src/course/ical.service.ts

```diff
@@ -95,7 +95,7 @@ export class IcalService {
   parseIcal(
     icalData: CalendarResponse,
     courseId: number,
-    testRegex = /\b^(OH|Hours)\b/,
+    testRegex = /\b^(Online OH)\b/,
   ): CreateOfficeHour {
     const icalDataValues: Array<CalendarComponent> = Object.values(icalData);
 
```

Comment:
Could you also uncomment the cron job in this file? It will allow the calendars to be scraped every night (since we only go 10 weeks out at a time)